autogpt_platform/backend/backend/notifications/email.py

```diff
@@ -62,7 +62,7 @@ def send_templated(
                 subject_template=template.subject_template,
                 content_template=template.body_template,
                 data=data,
-                unsubscribe_link="https://autogpt.com/unsubscribe",
+                unsubscribe_link="https://platform.agpt.co/profile/settings",
```

Comment:
shouldn't this be configurable?

Reply:
the other thing would be having user dedicated unsubscribe with tokens but that's out of scope for this pr